fresnel/geometry.py

```diff
@@ -183,6 +183,183 @@ def radius(self):
     def color(self):
         return util.array(self._geometry.getColorBuffer(), geom=self)
 
+
+class Box(Cylinder):
+    R""" Box geometry.
+
+    Generate a triclinic box shape.
+
+    Args:
+        scene (:py:class:`fresnel.Scene`): Add the geometry to this scene
+        box (`numpy.ndarray` or `array_like`): (``1x1, 1x3, or 1x6`` : ``float32``) Assumes 1x1 is cubic, 1x3 is orthorhombic, and 1x6 is triclinic.
+        radius (`float`): Radius of box edges.
+        color (`numpy.ndarray` or `array_like`): (``1x3`` : ``float32``) Color of box edges.
+
+    .. seealso::
+
+        :doc:`examples/01-Primitives/05-Box-geometry`
+            Tutorial: defining and setting box geometry properties
+        :doc:`examples/02-Advanced-topics/05-GSD-visualization`
+            Tutorial: Visualizing GSD files
+
+
+    Note:
+        The constructor arguments ``radius`` and ``color`` are optional. If you do not provide them,
+        they are initialized to 0.5 and black, respectively.
+
+    Note:
+        The Box class is constructed from Cylinders, which can be modified invidually. The convenience attributes ``box_radius`` and ``box_color`` can be used to easily modify the thickness and color of the box. If the individual cylinders are modified, the getter for the ``box_radius`` will return the radius of the 0th element cylinder.
+
+    Attributes:
+        points (:py:class:`fresnel.util.array`): Read or modify the start and end points of the cylinders.
+        radius (:py:class:`fresnel.util.array`): Read or modify the radii of the cylinders.
+        color (:py:class:`fresnel.util.array`): Read or modify the colors of the start and end points of the cylinders.
+        box (`tuple`): Read or modify the box parameters. Boxes will be converted from any acceptable box type to a tuple (Lx, Ly, Lz, xy, xz, yz). Changing the box will update the points used to generate the cylinders.
+        box_radius (`float`): Read or modify the radii of all the cylinders.
+        box_color (`tuple`): Read or modify the color of the box material.
+    """
+
+    def __init__(self,
+                 scene,
+                 box,
+                 radius=0.5,
```

Comment:
Should we name the constructor args `box_radius` and `box_color` to be consistent with the attributes? Or is it more convenient that the arg names are shorter?